tools/codegen/pkg/modelgen/model_test.go

```diff
@@ -34,14 +30,13 @@ func TestErrorInTemplate(t *testing.T) {
 		src           string
 		expectedError string
 	}{
-		{"testdata/MissingPackageName.proto", "package name missing"},
-		{"testdata/MissingTemplateNameExt.proto", "Contains only one of the following two options"},
-		{"testdata/MissingTemplateVarietyExt.proto", "Contains only one of the following two options"},
-		{"testdata/MissingBothRequiredExt.proto", "one proto file that has both extensions"},
-		{"testdata/MissingTypeMessage.proto", "message 'Type' not defined"},
-		{"testdata/MissingConstructorMessage.proto", "message 'Constructor' not defined"},
-		{"testdata/ReservedFieldInConstructor.proto", "proto:15: Constructor message must not contain the reserved filed name 'Name'"},
-		{"testdata/Proto2BadSyntax.proto", "Proto2BadSyntax.proto:3: Only proto3 template files are allowed."},
```

Comment:
This seems like an issue with merging (i did not intentionally strip those out). I'll investigate.